test/operations-conditions-1.js

```diff
@@ -0,0 +1,12 @@
+var assert = require("assert");
+function greaterNum(num1, num2) {
+  if (num1 > num2) {
```

Comment:
this condition should be simplified if u would take into account equal condition

Reply:
Do you mean rewrite condition? I didn't understand till the end this correction

Reply:
You do not care now about equal

For example, you could make one of condition inclusion with `<=` and in this case, another branch of `IF` could be simplified to simple else instead of `else if`